### ISSUE TITLE
train on multi gpus

### ISSUE THREAD

**Issue by liyu10000** [cvjena/cnn-models#25]
Hi,

I tried to finetune with your model. It works well on single gpu but not on multi gpus. Is it a inhere limitation or there is something I need to work around? Thanks.

**Comment by MarcelSimon** [comment]
Hi!

Which model are you using? Multi GPU shouldn't make a difference for VGG or 
AlexNet, but can make a difference for ResNet. For ResNet, please make sure 
that each GPU gets at least 8, better 16, images of each batch.

Another thing to take care of is the learning rate. It should be scale 
linearly with the batch size, which you probably scale when using multiple 
GPUs.

Am 19. Februar 2019 18:47:07 schrieb Li Yu <notifications@github.com>:

**Comment by liyu10000** [comment]
Thank you for your response. I finally found that there is something wrong with the environment. I tried to run it on another server and it worked like a charm.
